packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -759,7 +759,7 @@ Future<void> _writeIOSPluginRegistrant(FlutterProject project, List<Plugin> plug
   final List<Map<String, Object?>> iosPlugins = _extractPlatformMaps(methodChannelPlugins, IOSPlugin.kConfigKey);
   final Map<String, Object> context = <String, Object>{
     'os': 'ios',
-    'deploymentTarget': '9.0',
+    'deploymentTarget': '11.0',
```

Comment:
This is not used for published plugins, it's generated for app and modules that need to import plugins.